README.md

```diff
@@ -16,15 +16,26 @@ Please note that Ansible 2.9 and ansible-base 2.10 are no longer supported. If y
 
 ## External requirements
 
-Most modules and plugins require the [Docker SDK for Python](https://pypi.org/project/docker/) and Python 2.7 or newer. Python 2.6 is no longer supported; use community.docker 2.x.y if you need to use Python 2.6.
+Some modules and plugins require the Docker CLI programs or other external programs, some require the [Docker SDK for Python](https://pypi.org/project/docker/), and some use [requests](https://pypi.org/project/requests/) to directly talk to the Docker daemon's API. All modules and plugins require Python 2.7 or newer. Python 2.6 is no longer supported; use community.docker 2.x.y if you need to use Python 2.6.
```

Comment:
```suggestion
Some modules and plugins require Docker CLI, or other external, programs. Some require the [Docker SDK for Python](https://pypi.org/project/docker/) and some use [requests](https://pypi.org/project/requests/) to directly communicate with the Docker daemon API. All modules and plugins require Python 2.7 or later. Python 2.6 is no longer supported; use community.docker 2.x.y if you need to use Python 2.6.
```

Reply:
A few nits for structure and clarity. Also s/talk/communicate to avoid anthropomorphic language.